### DISCUSSION TITLE
[solved] MaterialDB ?

### DISCUSSION THREAD

**Discussion by wega-svg** [arkypita/LaserGRBL#2086] [Q&A]
Hello,
I'm using Windows 7 and I'm looking for the UserMaterialDB.
Where is their storage location?
With regards, Werner

**Comment by StuartB4** [comment]
![2](https://github.com/arkypita/LaserGRBL/assets/16980026/08f8009c-00b5-4a2f-b5e6-8af57de5d810)

You can open it wih a text editor to view it. 
![Untitled](https://github.com/arkypita/LaserGRBL/assets/16980026/9b5e222f-7d38-4603-aaf0-67e6a4dce7f4)

Also here.
![3](https://github.com/arkypita/LaserGRBL/assets/16980026/f61788ba-fa9f-483e-a811-f267688f5182)

**Reply by wega-svg** [reply]
Thank you for your quick response.
The "StandardMaterial.psh" file does not exist on my 32-bit system. Not even in the "Application Data" directory.
Can this file be downloaded from somewhere?
Greeting Werner

**Reply by StuartB4** [reply]
Not sure if you can get it on it's own but I've uploaded mine to my PCloud account, which is as it was when I installed the latest version of LaserGRBL (4.9.4). You don't need to have an account to get it, the link is below. 

https://u.pcloud.link/publink/show?code=XZiAIOVZBeSpMkcl0zyuyJRrXFRDM5Xm4Tcy


![4](https://github.com/arkypita/LaserGRBL/assets/16980026/3e08f545-58a4-417f-8738-aa06740e4d75)

**Comment by wega-svg** [comment]
I also installed the latest version of LaserGRBL (4.9.4), but the material database was not there.
Thank you for providing the database. Now I have them too.
Thank you and best regards Werner

**Reply by StuartB4** [reply]
You're very welcome.
